Airports/EGVA/Sids.txt

```diff
@@ -0,0 +1,4 @@
+SID:EGVA:09:GIBMI1A:GIBMI
```

Comment:
Do these have initial climbs? If so, presumably need to be set in the plugin API

Reply:
Initial climb is coordinated with LON MIL (or West on VATSIM I suppose) I believe, same as Brize would be

Charts give a restriction of at or below FL100 by the end points so we could use FL100?

Reply:
Initial climbs have been added to the plugin, thanks to Andy for making the changes